test/cli/run/as-node.test.ts

```diff
@@ -90,11 +89,11 @@ describe("fake node cli", () => {
 
   test("process args work", () => {
     const temp = tempDirWithFiles("fake-node", {
-      "index.js": "console.log(JSON.stringify(process.argv))",
+      "index.js": "console.log(JSON.stringify(process.argv.slice(1)))",
```

Comment:
we do need to test that process.argv[0] is as expected

Reply:
I included in the commit message why this was done